ext/process/lib.rs

```diff
@@ -997,6 +1055,11 @@ fn op_spawn_kill(
 }
 
 mod deprecated {
+  #[cfg(windows)]
+  use deno_subprocess_windows::Child;
+  #[cfg(not(windows))]
+  use tokio::process::Child;
```

Comment:
I believe `Deno.run` (which this module implements) is not used in Node compat layer and doesn't necessarily needs to be updated to use the new crate - or is it just simpler for you to do so?